salt/iiif/init.sls

```diff
@@ -0,0 +1,224 @@
+get-loris:
+    docker_image.present:
+        - name: elifesciences/loris:latest
```

Comment:
https://alfred.elifesciences.org/job/process/job/process-iiif/ doesn't modify the build vars, so I think `latest` or another pinned tag is the best we can do.

Reply:
Best we can do for now. 

IIIF has a 3.0.0 released about a month ago with a lot of bug fixes and things in it. It could be an opportunity to introduce more nuanced tags